contracts/epochs/FlowEpoch.cdc

```diff
@@ -674,24 +674,36 @@ pub contract FlowEpoch {
 
     /// Borrow a reference to the FlowIDTableStaking Admin resource
     access(contract) fun borrowStakingAdmin(): &FlowIDTableStaking.Admin {
```

Comment:
```suggestion
    access(contract) fun borrowStakingAdmin(): &FlowIDTableStaking.Admin{FlowIDTableStaking.EpochOperations} {
```

Based on the error in the CI logs, I'm guessing this is causing the `flow-go` failure you mentioned? If so, `borrowClusterQCAdmin` would need a similar update.

> panic: failed to deploy Epoch contract: [Error Code: 1101] error caused by: 1 error occurred:
> 	* transaction execute failed: [Error Code: 1101] cadence runtime error: Execution failed:
> error: cannot deploy invalid contract
>   --> 6f7d6e0ffbb786cbf619254e6b3f9aa1b64d45896a6f8385480ef9fcdf10e62f:23:1
>    |
> 23 | 	serviceAccount.contracts.add(
>    | 	^
> 
> error: mismatched types
>    --> f8d6e0586b0a20c7.FlowEpoch:684:15
>     |
> 684 |         return adminRef
>     |                ^^^^^^^^ expected `&FlowIDTableStaking.Admin`, got `&FlowIDTableStaking.Admin{FlowIDTableStaking.EpochOperations}`